core/query/annotated.go

```diff
@@ -34,7 +34,7 @@ type AnnotatedInput struct {
 	Amount          uint64             `json:"amount"`
 	IssuanceProgram chainjson.HexBytes `json:"issuance_program,omitempty"`
 	ControlProgram  chainjson.HexBytes `json:"control_program,omitempty"`
-	SpentOutput     *SpentOutput       `json:"spent_output,omitempty"`
```

Comment:
After thinking about it some, I think traversal of the transaction history is generically useful, and would rather not lose the ability to retrieve prevouts and/or their containing transactions. If it remains technically possible to do this during block processing, I think we should try to preserve the transaction ID and index annotations.

One possible outcome is the following:

```
{
  "spent_output": {
    "id": <output_id>,
    "transaction_id": <transaction_id>,
    "position": <position in transaction>
  }
}
```